bigtable/metricscaler/metricscaler.py

```diff
@@ -45,6 +46,25 @@ def get_cpu_load():
     # [END bigtable_cpu]
 
 
+def get_storage_utilization():
+    """Returns the most recent Cloud Bigtable storage utilization measurement.
+
+    Returns:
+          float: The most recent Cloud Bigtable storage utilization metric
+    """
+    # [START bigtable_metric_scaler_storage_utilization]
+    client = monitoring_v3.MetricServiceClient()
+    utilization_query = query.Query(client,
+                                    project=PROJECT,
+                                    metric_type='bigtable.googleapis.com/'
+                                                'cluster/storage_utilization',
+                                    minutes=5)
+    time_series = list(utilization_query)
```

Comment:
If we only read the first entry, we can avoid processing the full iterable. I have 4 entries when I run locally.
```suggestion
    time_series = next(utilization_query.iter())
```